src/js/components/MapPanelBookmarks.js

```diff
@@ -103,8 +104,13 @@ export default class MapPanelBookmarks extends React.Component {
      * Delete all bookmarks
      */
     onClickDeleteBookmarks () {
-        const modal = new Modal('Are you sure you want to clear your bookmarks? This cannot be undone.', bookmarks.clearData);
-        modal.show();
+        ReactDOM.render(
```

Comment:
It's also temporary - I _believe_ you can even pass a component in as state, so when the entire application is ported we can try to set modals in state properties instead of calling ReactDOM.render manually.